src/sentry/static/sentry/app/views/organizationHealth/errors.jsx

```diff
@@ -1,9 +1,342 @@
+import {Box, Flex} from 'grid-emotion';
+import PropTypes from 'prop-types';
 import React from 'react';
+import styled from 'react-emotion';
+
+import {addErrorMessage} from 'app/actionCreators/indicator';
+import {t} from 'app/locale';
+import AreaChart from 'app/components/charts/areaChart';
+import Count from 'app/components/count';
+import IdBadge from 'app/components/idBadge';
+import PercentageBarChart from 'app/components/charts/percentageBarChart';
+import PieChart from 'app/components/charts/pieChart';
+import SentryTypes from 'app/sentryTypes';
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+import space from 'app/styles/space';
+import withApi from 'app/utils/withApi';
+
+import Header from './styles/header';
+import HealthPanelChart from './styles/healthPanelChart';
+import HealthRequest from './util/healthRequest';
+import HealthTableChart from './styles/healthTableChart';
+import withHealth from './util/withHealth';
 
 class OrganizationHealthErrors extends React.Component {
+  static propTypes = {
+    actions: PropTypes.object,
+    organization: SentryTypes.Organization,
+  };
+
+  handleSetSpecifier = (tag, value) => {
+    this.props.actions.setSpecifier(tag, value);
+  };
+
   render() {
-    return <div>Errors</div>;
+    let {organization} = this.props;
+
+    return (
+      <React.Fragment>
+        <Flex justify="space-between">
+          <Header>{t('Errors')}</Header>
+        </Flex>
+
+        <Flex>
+          <HealthRequest
+            tag="error.handled"
+            includeTimeseries
+            interval="1d"
+            showLoading
+            getCategory={value => (value ? 'Handled' : 'Crash')}
+          >
+            {({timeseriesData}) => {
+              return (
+                <HealthPanelChart
+                  height={200}
+                  title={t('Errors')}
+                  series={timeseriesData}
+                >
+                  {props => <AreaChart {...props} />}
+                </HealthPanelChart>
+              );
+            }}
+          </HealthRequest>
+
+          <HealthRequest tag="user" showLoading includeTop includeTimeseries={false}>
+            {({originalTagData: originalData, tag}) => (
+              <HealthTableChart
+                headers={[t('Most Impacted')]}
+                data={originalData.map(row => [row, row])}
+                widths={[null, 120]}
+                getValue={item => (typeof item === 'number' ? item : item && item.count)}
+                renderHeaderCell={({getValue, value, columnIndex}) => {
+                  return typeof value === 'string' ? (
+                    value
+                  ) : (
+                    <div>
+                      <IdBadge
+                        user={value[tag].value}
+                        displayName={
+                          value[tag] && value[tag].value && value[tag].value.label
```

Comment:
ouch, can we drop some of these checks if not needed, or ensure some of these types are consistent? e.g. Is there a valid reason why every value can't have a label? (I assume the first one isn't required in any case since it's already used on line 77)

Reply:
https://github.com/getsentry/sentry/pull/9658/commits/67f44205e615c8856f1e37740003b1c846e3b5b3